### PR TITLE
LPS-190069 Update Country/Region codes as for latest ISO list

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPS-190069

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

File: modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/upgrade/v1_0_1/CountryRegionUpgradeProcess.java [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+
+			_updateRegions(country);
+		}
+	}
+
+	private void _updateData(
+			long newRegionId, long oldRegionId, String tableName)
+		throws SQLException {
+
+		PreparedStatement preparedStatement = connection.prepareStatement(
```
Comment (andrea-ale-sbarra) [inline modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/upgrade/v1_0_1/CountryRegionUpgradeProcess.java:153]: please use try to autoclose the preparedStatement

File: modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/upgrade/v1_0_1/CountryRegionUpgradeProcess.java [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+
+		preparedStatement.executeUpdate();
+	}
+
+	private void _updateData(
+			String countryA2, String newRegionCode, String oldRegionCode,
+			String tableName)
+		throws SQLException {
+
+		PreparedStatement preparedStatement = connection.prepareStatement(
```
Comment (andrea-ale-sbarra) [inline modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/upgrade/v1_0_1/CountryRegionUpgradeProcess.java:168]: try

File: modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/upgrade/v1_0_1/CountryRegionUpgradeProcess.java [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+		preparedStatement.executeUpdate();
+	}
+
+	private void _updateRegion(
+			String countryA2, String newRegionCode, String oldRegionCode,
+			String oldRegionName)
+		throws Exception {
+
+		try (PreparedStatement preparedStatement = connection.prepareStatement(
+				SQLTransformer.transform(
```
Comment (andrea-ale-sbarra) [inline modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/upgrade/v1_0_1/CountryRegionUpgradeProcess.java:186]: There is no need to use the SQLTransformer here

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
Just little changes, overall looks good.
Thank you!

**Comment by smottal** [comment]
ci:test:sf

File: modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/upgrade/v1_0_1/CountryRegionUpgradeProcess.java [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+					if (localizationsJSONObject == null) {
+						Map<String, String> titleMap = new HashMap<>();
+
+						for (Locale locale :
+								LanguageUtil.getCompanyAvailableLocales(
+									country.getCompanyId())) {
+
+							titleMap.put(
+								LanguageUtil.getLanguageId(locale),
+								region.getName());
```
Comment (andrea-ale-sbarra) [inline modules/apps/address/address-impl/src/main/java/com/liferay/address/internal/upgrade/v1_0_1/CountryRegionUpgradeProcess.java:271]: can you try to use the builder here?

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:upgrade

**Comment by smottal** [comment]
ci:test:upgrade

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**